app/src/main/java/org/projectbuendia/client/models/Obs.java

```diff
@@ -28,62 +27,77 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /** A simple bean class representing an observation with localized names and values. */
-public final class ObsValue implements Comparable<ObsValue> {
-    public static final Comparator<ObsValue> BY_OBS_TIME = new Comparator<ObsValue>() {
-        @Override public int compare(ObsValue left, ObsValue right) {
-            return left.obsTime.compareTo(right.obsTime);
-        }
-    };
-
+// TODO: Make ObsPoint a member of Obs; change the structure of Obs to be simply:
+// { final @Nonnull String uuid; String name; final @Nonnull ObsPoint point; } then delete
+// getObsPoint(), getObsValue(), compareTo(), getTypeOrdering(), getCodedValueOrdering().
```

Comment:
Yes, it's intentional that ObsPoint compares by time whereas ObsValue compares by value/severity.  Both kinds of sorting are useful, and these seem like reasonable defaults to me, especially when you think of it as:
  - ObsPoint = (x, y), which is sorted by x
  - ObsValue = y, which is sorted by y